src/Microsoft.ApiDesignGuidelines.Analyzers/Core/OverrideEqualsOnOverloadingOperatorEquals.Fixer.cs

```diff
@@ -27,10 +23,57 @@ public sealed override FixAllProvider GetFixAllProvider()
         }
 
         public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
```

Comment:
Sorry, let me be more clear.  I agree with you that the fixer should be more like:

```
------------------------------------------
| Add .Equals override                   |
| Add .Equals and .GetHashCode overrides | 
------------------------------------------
```